cluster/v1/types.go

```diff
@@ -82,6 +86,43 @@ type ClientConfig struct {
 	CABundle []byte `json:"caBundle,omitempty"`
 }
 
+// The managed cluster this Taint is attached to has the "effect" on
+// any placement that does not tolerate the Taint.
+type Taint struct {
+	// The taint key to be applied to a cluster. e.g. bar or foo.example.com/bar.
+	// The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
+	// +required
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Pattern=`^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*/)?(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])$`
+	// +kubebuilder:validation:MaxLength=316
+	Key string `json:"key"`
+	// The taint value corresponding to the taint key.
```

Comment:
It commonly start with "Value defines ..."

Reply:
Updated.